backend/curation/api/v1/curation/collections/common.py

```diff
@@ -4,6 +4,7 @@
 from uuid import UUID
 
 from backend.common.corpora_config import CorporaConfig
+from backend.common.feature_flag import FeatureFlagService, FeatureFlagValues
```

Comment:
Is there ever a situation where `FeatureFlagService` and not `FeatureFlagValues` would be imported? `FeatureFlagValues` could be a property  of `FeatureFlagService` to simplify or maybe something similar to.

Reply:
I don't think so, but there are cases where you'd want to import `FeatureFlagValues` and not `FeatureFlagService`.